repos/exawind/packages/hypre/package.py

```diff
@@ -4,6 +4,18 @@
 
 class Hypre(bHypre):
 
+    phases = ['autoreconf', 'distclean', 'configure', 'clean', 'build', 'install']
+
+    def distclean(self, spec, prefix):
+        with working_dir('src'):
+            if os.environ.get('SPACK_MANAGER_CLEAN_HYPRE') is not None:
```

Comment:
```suggestion
            if 'SPACK_MANAGER_CLEAN_HYPRE' in os.environ:
```
This is a little cleaner way to check for a dictionary entry in python